The Downlaod ZIP action can not be performed using the keyboard

sofyakurilova opened the pull request:
Stop propagating event to eliminate the error
"+" fix tooltip
![BD6pc5LTD3LVc2R](https://github.com/google/testrun/assets/22660354/f36d42e6-39ce-44af-8ac0-9ca61b63071d)

jboddey commented:
I have fixed an issue with the API where the export endpoint had not been updated to accept POST requests. However, the UI is trying to interpret the response as JSON instead of downloading a ZIP file @sofyakurilova

sofyakurilova commented:
> I have fixed an issue with the API where the export endpoint had not been updated to accept POST requests. However, the UI is trying to interpret the response as JSON instead of downloading a ZIP file @sofyakurilova

Fixed http://screencast/cast/NTUwNjI2MTYzMzIwNDIyNHxjN2MxY2FkNy0yOQ
